models/balancer/arbitrum/balancer_arbitrum_schema.yml

```diff
@@ -374,6 +374,41 @@ models:
       - name: tx_to
       - name: evt_index
 
+  - name: balancer_v2_arbitrum_protocol_revenue
+    meta:
+      blockchain: arbitrum
+      project: balancer_v2
+      contributors: viniabussafi
+    config:
+      tags: ['arbitrum', 'revenue', 'fees']
+    description: >
+      Daily swap Fees and Yield revenues by pool on Balancer, an automated portfolio manager and trading platform.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - day
+            - pool_id
+            - token_address
+    columns:
+      - name: day
+      - name: pool_id
+      - name: pool_address
+      - name: pool_symbol
+      - name: blockchain
+      - name: token_address
+      - name: token_symbol
+        description: "Symbol of the token"
+      - name: token_balance_raw
+        description: "Raw amount of revenues of the token in the pool"
+      - name: token_balance
+        description: "Raw amount of revenues of the token in the pool"
```

Comment:
you called this `token_amount_raw` and `token_amount` in the SQL - again, we may remove it